### PR TITLE
Manual page for the Depth Layer

### PR DESCRIPTION
**Pull request by mvaligursky**
Fixes https://github.com/playcanvas/developer.playcanvas.com/issues/390

new page:
Screenshot 2022-06-21 at 11 51 57

reference from Layers page:
Screenshot 2022-06-20 at 12 20 30

new section on particles manual:
Screenshot 2022-06-21 at 12 09 39

### CODE REVIEW

File: content/en/user-manual/graphics/layers/index.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ PlayCanvas applications are created with a set of default layers. You should lea
 
 ![Default Layers][1]
 
-1. **World (Opaque)** - Used to render components that are not transparent and most opaque component meshes.
-1. **Depth (Opaque)** - The depth layer is for advanced users only.
+1. **World (Opaque)** - Used to render components that are not transparent and most opaque component meshes.
+1. **Depth (Opaque)** - Used to capture the color or the depth buffer of the scene, see [Depth Layer][depth-layer].
```
Comment (yaustar) [inline content/en/user-manual/graphics/layers/index.md:54]: Minor: Keep the numbering system for links if the page already uses them or change all of them so we don't mix and match please

File: content/en/user-manual/graphics/cameras/depth-layer.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Depth Layer
+layout: usermanual-page.hbs
+position: 2
+---
+
+Some rendering techniques require access to the depth or the color buffer of the scene for a specific camera. The Depth Layer is a special layer, which can be added to the `layers` property of a camera, to define at which point during the rendering, the depth or the color buffer is captured. The captured buffers can then be used in the following layers of the camera.
```
Comment (yaustar) [inline content/en/user-manual/graphics/cameras/depth-layer.md:7]: This makes it sound like adding the depth layer to the camera decides when the depth is captured to a buffer. Isn't it defined by the render layer order? Or is this more about camera stacking?

File: content/en/user-manual/graphics/cameras/depth-layer.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Typically, those buffers are captured after all opaque layers are rendered, and can be used in following transparent layers or post-processing.
+
+Additionally, to capture these buffers, the capture needs to be enabled on a CameraComponent from the script:
+- [```requestSceneColorMap```][0] to request a color map
+- [```requestSceneDepthMap```][1] to request a depth map
```
Comment (yaustar) [inline content/en/user-manual/graphics/cameras/depth-layer.md:13]: Is there a performance cost in the capture?

**Review by willeastcott — APPROVED** [review]

File: content/en/user-manual/graphics/layers/index.md (hunk 1 above)
Comment (mvaligursky) [reply content/en/user-manual/graphics/layers/index.md:54]: I think it's a lot more convenient to move to a name based, even one by one? Does it create some problems?

Comment (willeastcott) [reply content/en/user-manual/graphics/layers/index.md:54]: I think I'm with @yaustar on this - it's pretty messy to mix the two styles. So I'd recommend switching to `7` please.

File: content/en/user-manual/graphics/cameras/depth-layer.md (hunk 2 above)
Comment (mvaligursky) [reply content/en/user-manual/graphics/cameras/depth-layer.md]: makes sense, adjusted, thanks

**Comment by mvaligursky** [comment]
> Do we need to add a note to particle effects and depth softening?

good idea, done

**Review by yaustar — APPROVED** [review]

**Pull request merged**